oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileNodeStoreBuilder.java

```diff
@@ -319,23 +340,26 @@ private List<File> getFiles(File sortedDir) {
         return null;
```

Comment:
With regards to my comment, if we choose to go with the custom data class, can we return an `optional`, or `emptyMap()`? I'd prefer us to not return null wherever possible.